### DISCUSSION TITLE
npx create-next-app@latest creating broke app

### DISCUSSION THREAD

**Discussion by SteveRosam** [vercel/next.js#59007] [Help]
### Summary

I run `npx create-next-app@latest` and go with the default options then run `next dev`.
I get the error: 

TypeError: Cannot read properties of null (reading 'useReducer')

Is this normal? new to NextJS/react

**Comment by icyJoseph** [comment]
Can't reproduce. 

What have you done with the application? If you are using `App router` , you need to use `"use client"` at the top of files that use React hooks.

- https://react.dev/reference/react/use-client
- https://nextjs.org/docs/app/building-your-application/rendering/client-components#using-client-components-in-nextjs
